test_schemas/en/test_address.json

```diff
@@ -60,6 +60,44 @@
                                     }
                                 ]
                             }
+                        },
+                        {
+                            "type": "Question",
+                            "id": "address-confirmation",
+                            "question": {
+                                "type": "General",
```

Comment:
Could be a confirmation question

Reply:
I reckon it will be ok, considering its a test schema